contrib/database/sql/sql_test.go

```diff
@@ -271,6 +271,56 @@ func TestOpenOptions(t *testing.T) {
 		s0 := spans[0]
 		assert.Equal(t, "register-override", s0.Tag(ext.ServiceName))
 	})
+
+	t.Run("WithDBStats", func(t *testing.T) {
+		Register(driverName, &pq.Driver{})
+		defer unregister(driverName)
+		_, err := Open(driverName, dsn, WithDBStats())
+		require.NoError(t, err)
+
+		var tg statsdtest.TestStatsdClient
+		sc := internal.NewStatsCarrier(&tg)
+		sc.Start()
+		defer sc.Stop()
+		globalconfig.SetStatsCarrier(sc)
+
+		// check that within 13s, at least one round of `pollDBStats` has completed
+		deadline := time.Now().Add(13 * time.Second)
```

Comment:
Since you're in the same package, could you just change `interval` to be a much smaller number for this test (e.g. 5 ms), then you don't have to wait 13 seconds for this to complete?